dev/source/docs/building-px4-with-qtcreator.rst

```diff
@@ -0,0 +1,233 @@
+In Linux for example, that is achieved by:
+
+.. code-block:: bash
+
+	ln -s ./generate_ardupilot_project.sh ./.git/hooks/post-merge
+	ln -s ./generate_ardupilot_project.sh ./.git/hooks/post-checkout 	
+
+Build the project
+=================
+This section discusses how to build the code in Qt Creator.
+
+#. Click on ``Projects`` on the left pane and make sure that you are in the 
+   ``Build & Run`` tab page. 	
+
+#. Click ``Manage Kits`` in the topleft corner.
+
+#. First click on the ``Compilers`` tab page and then "Add" on the right hand side of the compilers list.
+   Choose an easily recognisable name for your compiler and make sure the Compiler and Make path are 
+   referring to the executables of the PX4 toolchain (Windows) or the downloaded gcc-arm cross-compiler
+   (Linux). Also choose "GCC" as the Error parser.
+   
+   .. image:: ../images/QtCreator_ManageCompilers.png
+	 :target: ../_images/QtCreator_ManageCompilers.png     
+
+#. Then click on the ``Kits`` tab page. Click ``Add`` on the right hand side.
+
+   Choose an easily recognisable name for your build kit and make sure you fill in the
+   proper compiler (the one you just added) and the debugger inside the PX4 toolchain.
+
+   .. image:: ../images/QtCreator_ManageKits.png
+	  :target: ../_images/QtCreator_ManageKits.png   
+
+#. Click apply.
+
+#. Back on the ``Projects`` page, click ``Add Kit`` and choose the Build Kit you just added.
+
+#. Now you have one "Build Configuration" called "Default". You can make as many Build Configurations
+   as you want, but we'll take the ArduCopter build as an example for now. 
+   Next to ``Edit build configuration``, click on ``Add`` and choose ``Clone Selected`` in the 
+   drop down menu. Pick a name (e.g. "Copter").
+   
+#. Click on the ``Details`` of the ``Build Steps`` and type "px4-v2 -j2" as ``Make arguments``. 
+   Deselect the ``Targets``.
+
+#. Click on the ``Details`` of the ``Clean Steps`` and type "px4-clean" as ``Make arguments``.
+   Deselect the ``Targets``.
+
+   .. image:: ../images/QtCreator_Target_Project_Settings.png
+      :target: ../_images/QtCreator_Target_Project_Settings.png   	
+	
+#. You can make other build configurations for e.g. ArduPlane in the same way. You can quickly switch
+   between "Build Configurations" by clicking the logo just above the ``Run`` icon (the green arrow) on 
+   the left pane. 	
+	
+#. You can now remove the MSVC or standard GCC build kit (click on the down arrow on the kit itself and 
+   choose ``Remove Kit``)
+   
+#. You're now ready to build the code. Click on ``Edit`` in the left pane to edit the code and browse
+   through the project. Click ``Build project-name`` in the ``Build`` menu (or Ctrl+B) to build the code.    
+
+Qt Creator editor settings
+==========================		
+#. It is handy to set the tab and indentation sizes to be used while editing. Click on the ``Tools`` menu
```

Comment:
Again, use bold for button/menu presses. It is important to be consistent - here you are using ``Tools`` before you were using "Tools".